File: Domain/Entities/Question.cs
using Domain.Enums;

namespace Domain.Entities;

public class Question
{
    public int Id { get; set; }
    public QuestionType Type { get; set; }
    public string Title { get; set; }
    public bool Required { get; set; }
    public string? AnswerChoices { get; set; }
    public Event Event { get; set; }
    public int EventId { get; set; }
    public List<Answer> Answers { get; set; } = new();
}

File: mobile/lib/domain/models/question.dart
import 'package:project/domain/models/question_type.dart';

class Question {
  final int id;
  final String question;
  final List<String> options;
  final QuestionType questionType;

  Question({
    required this.id,
    required this.question,
    required this.options,
    required this.questionType,
  });
}
